metadata-ingestion-modules/airflow-plugin/setup.py

```diff
@@ -24,9 +24,9 @@ def get_long_description():
 
 base_requirements = {
     f"acryl-datahub[datahub-rest]{_self_pin}",
-    # We require Airflow 2.3.x at minimum, since we need the new DAG listener API.
+    # We require Airflow 2.7.x at minimum, since we need the new DAG listener API.
```

Comment:
this comment is a bit inaccurate now - the DAG listener API was added in 2.3. We required 2.7+ for a different reason, which should be listed here

Reply:
it's in metadata-ingestion-modules/airflow-plugin/tests